GUIs/ScriptRunnerPyQt5_GUI/ScriptGUIrunner.py

```diff
@@ -331,64 +330,27 @@ class Mainstart(QMainWindow, Ui_MainWindow):
 
     def threader(self, com):
         try:
-            threader = Thread(target=com)
+            threader = Thread(target = com)
             threader.start()
         except Exception as e:
             print(e)
             print('Could not start thread')
 
-    def __init__(self, parent=None):
+    def __init__(self, parent = None):
         super(Mainstart, self).__init__(parent)
         print('Script Runner has started')
-        # self.size().setHeight(675)
-        # self.size().setWidth(860)
-        # a = self.size()
-        # print(a)
-        # print(a.height())
-        # print(a.width())
-        # screen = app.primaryScreen()
-        # print('Screen: %s' % screen.name())
-        # size = screen.size()
-        # print('Screen Size: %d x %d' % (size.width(), size.height()))
-        # rect = screen.availableGeometry()
-        # print('Available: %d x %d' % (rect.width(), rect.height()))
-        # hScaled = size.height()/1080
-        # print(hScaled)
-        # hScaled = hScaled*610
-        # print(hScaled)
-
-        config_name = 'config.ini'
+        self.setFixedSize(860, 675)
+        self.setupUi(self)
+        self.setWindowIcon(QtGui.QIcon(':/Pictures/images/cup2.png'))
+        config_name2 = 'config.ini'
         if getattr(sys, 'frozen', False):
-            application_Path = os.path.dirname(sys.executable)
+            application_Path2 = os.path.dirname(sys.executable)
         elif __file__:
-            application_Path = os.path.dirname(__file__)
-        config_path = os.path.join(application_Path, config_name)
-        print(config_path)
-        variableCheck = Path(config_path)
-
-        if variableCheck.is_file():
-            config = configparser.ConfigParser()
-            config.read(variableCheck)
-            # x = config.get('Services', 'ssh')
-            # print(x)
-            print('Configuration file has been loaded...')
-
-            QMainWindow.__init__(self)
-            self.setupUi(self)
-            # self.setFixedWidth(848)
-            # self.setFixedHeight(603)
-            # self.setFixedSize(850, 640)
-            self.setWindowIcon(QtGui.QIcon(':/Pictures/images/cup2.png'))
-            self.mmfuncassign(variableCheck)
-        else:
-            print('Ello, you have some configurations to do!')
-
-            def firstCONF():
-                widget = fconfStart()
-                widget.exec_()
-
-            self.threader(firstCONF())
+            application_Path2 = os.path.dirname(__file__)
+        config_path2 = os.path.join(application_Path2, config_name2)
```

Comment:
Codacy found an issue: ['os' may be undefined, or defined from star imports: PyUIs.firstconf, scripFunc.scripLINUXONLY, scripFunc.scripUNIMULTI, scripFunc.scripWINONLY, threading (F405)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927419)